tests/ui/suggestions/remove-brackets-range-issue-147944.rs

```diff
@@ -0,0 +1,35 @@
+fn main() {
+    let mut vec = vec![1, 2, 3, 4, 5];
+    vec.drain([..3]);
+    //~^ ERROR: the trait bound `[RangeTo<{integer}>; 1]: RangeBounds<usize>` is not satisfied [E0277]
+    //~| NOTE: the trait `RangeBounds<usize>` is not implemented for `[RangeTo<{integer}>; 1]`
+    //~| NOTE: required by a bound introduced by this call
+    //~| NOTE: required by a bound in `Vec::<T, A>::drain`
```

Comment:
IIUC you can remove all the `NOTE` annotations in the file, they're not really related to what you're testing.

I'd rather prefer you add a `SUGGESTION` annotation here.